lte/gateway/c/core/oai/test/amf/test_amf_encode_decode.cpp

```diff
@@ -880,6 +907,372 @@ TEST(test_dnn, test_amf_validate_dnn) {
   EXPECT_TRUE(rc == RETURNok);
 }
 
+class AmfUeContextTestServiceRequestProc : public ::testing::Test {
+ protected:
+#define MCC_DIGIT1 2
+#define MCC_DIGIT2 2
+#define MCC_DIGIT3 2
+#define MNC_DIGIT1 4
+#define MNC_DIGIT2 5
+#define MNC_DIGIT3 6
+#define IMSI64 222456000000101
+#define M_TMSI 0X212e5025
+#define AMF_SET_ID 1
+#define AMF_POINTER 0
+#define AMF_REGION_ID 1
+#define AMF_TAC 0x03
```

Comment:
looks redundant to me. are they used anywhere else other than initializing the class variables?

Reply:
used in testcases